locale/en_US/common.po

```diff
@@ -300,6 +300,9 @@ msgstr "Declined"
 msgid "common.delete"
 msgstr "Delete"
 
+msgid "common.publication.authorsString"
+msgstr "Publication Authors Display"
```

Comment:
This can be removed after my changes to `contributtorsListPanel` are merged.

Reply:
Removed